elf_reader_test.go

```diff
@@ -329,7 +329,7 @@ func TestGetProgType(t *testing.T) {
 	}
 
 	for _, tc := range testcases {
-		pt, at, to := getProgType(tc.section)
+		pt, at, _, to := getProgType(tc.section)
```

Comment:
Can you add a test case for this? Also assert that flags are 0 for regular sections.